src/pages/_app.tsx

```diff
@@ -17,15 +17,26 @@ const App = ({ Component, pageProps }: AppProps): ReactNode => {
 
             {/* // Import all top-level scripts here: https://nextjs.org/docs/messages/no-script-in-document-page */}
 
-            {/* Triblio "Webpage Personalization" for Sales/Marketing */}
+            {/* Triblio "Webpage Personalization" */}
             <Script
-                data-cfasync="false"
                 type="text/javascript"
                 src="https://tribl.io/h.js?orgId=Yee6bMKj7QSARqAePdE8"
+                strategy="beforeInteractive"
+            />
+            
+            {/* Triblio "Analytics and Overlay Cards" */}
+            <Script
+                type="text/javascript"
+                src="https://tribl.io/footer.js?orgId=Yee6bMKj7QSARqAePdE8"
```

Comment:
The Triblio footer script (analytics and overlays) doesn't appear to load on pages: 

<img width="800" alt="Screen Shot 2022-05-25 at 1 07 38 PM" src="https://user-images.githubusercontent.com/60713139/170335622-da22f262-fd57-4b2c-bb58-d46026aca83e.png">

Reply:
Hmm, I'm not able to replicate. I see the scripts loaded in Sources and Network. I also see the `TriblioAnalyticsObject` in the console.

![Screen Shot 2022-05-25 at 3 14 49 PM](https://user-images.githubusercontent.com/1733936/170350624-5075f308-7456-4d72-bfe3-ac247045c81e.png)
![Screen Shot 2022-05-25 at 3 15 04 PM](https://user-images.githubusercontent.com/1733936/170350625-d966ec47-ca07-4867-a90e-dbaa8f0edbb4.png)
![Screen Shot 2022-05-25 at 3 15 25 PM](https://user-images.githubusercontent.com/1733936/170350626-886eca3a-c9b4-4d9c-a456-9b05cc8e8a7d.png)
![Screen Shot 2022-05-25 at 3 15 40 PM](https://user-images.githubusercontent.com/1733936/170350627-adf774ec-375c-493d-a490-2c16e1eb464c.png)
![Screen Shot 2022-05-25 at 3 16 43 PM](https://user-images.githubusercontent.com/1733936/170350630-ea557ed3-c58b-4475-a5ea-40d97dc55992.png)

Reply:
As noted in my other comment, this now loads in the body as expected now that you added the IDs!